specs/core/0_beacon-chain.md

```diff
@@ -161,6 +161,7 @@ The following values are (non-configurable) constants used throughout the specif
 | `BASE_REWARDS_PER_EPOCH` | `5` |
 | `DEPOSIT_CONTRACT_TREE_DEPTH` | `2**5` (= 32) |
 | `SECONDS_PER_DAY` | `86400` |
+| `JUSTIFICATION_BITS_LENGTH` | `4` |
```

Comment:
Went back and forth on this one a few days ago. You _can_ technically make it any value `>=4` if for some reason you wanted more historic info around

Reply:
That said, don't really care..

Reply:
Oh right, technically configurable. I'd say put it in constants to not confuse people.